samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/RepublishAssessmentListener.java

```diff
@@ -15,11 +15,8 @@
  */
 package org.sakaiproject.tool.assessment.ui.listener.author;
 
-import java.util.Date;
```

Comment:
Revert this changes please, can make compatibility harder. I know they are unused and improves the overall quality, but will make cherry-picks harder.

Reply:
Ahhh, it's moved logic, its fine thenm